queries/c/highlights.scm

```diff
@@ -170,6 +172,28 @@
 (preproc_def (preproc_arg) @constant.builtin
   (#any-of? @constant.builtin "stderr" "stdin" "stdout"))
 
+((identifier) @variable.builtin
+  (#any-of? @variable.builtin
```

Comment:
These are technically constants. :)

```suggestion
((identifier) @constant.builtin
  (#any-of? @constant.builtin
```

Reply:
it's really predefined macros, but, yeah constant works (`@macro.builtin` would be nice but it's too nice imo)